src/ng2-auto-complete.component.ts

```diff
@@ -5,23 +5,24 @@ import {
   Output,
   OnInit,
   ViewEncapsulation,
-  EventEmitter
+  EventEmitter,
+  ViewChild
 } from "@angular/core";
-import { Ng2AutoComplete } from "./ng2-auto-complete";
+import {Ng2AutoComplete} from "./ng2-auto-complete";
 
```

Comment:
I'll update them all. Angular team has no spaces around their imports in their code: 

But their style guide uses spaces very conflicting. If the communities choice is spaces i can update.